scripts/template_upload_rhevm.py

```diff
@@ -130,12 +130,12 @@ def template_from_ova(api, username, password, rhevip, edomain, ovaname, ssh_cli
         print("RHEVM: Skipping this step. Attempting to continue...")
         return
     command = ['rhevm-image-uploader']
-    command.append("-u %s" % username)
-    command.append("-p %s" % password)
-    command.append("-r %s:443" % rhevip)
-    command.append("-N %s" % TEMP_TMP_NAME)
-    command.append("-e %s" % edomain)
-    command.append("upload %s" % ovaname)
+    command.append("-u {}".format(username))
```

Comment:
as far as i can tell this could potentially be turned into a beautiful list instead of a long long line up of append statements